samples/sid_end_device/Kconfig.sysbuild

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_DEFAULT_BLUETOOTH
+	default y if (("$(BOARD)" = "nrf5340dk_nrf5340_cpuapp") || ("$(BOARD)" = "thingy53_nrf5340_cpuapp"))
```

Comment:
do not check board names, use Kconfig symbols

Reply:
squashed